src/logic/operate.js

```
@@ -0,0 +1,17 @@
+import Big from 'big.js';
+
+const Operate = (numberOne, numberTwo, operation) => {
+  const x1 = Big(numberOne);
+  const x2 = Big(numberTwo);
+
+  if (operation === '+') x1.plus(x2);
```

- [x] [OPTIONAL] Since you are always testing the value of the `operation` param you could use a [switch case statement](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/switch) instead of multiple ifs.

- [x] [OPTIONAL] What if the operation is none of the recognized valid strings? You could throw an error with a custom message about it.

---

Here the operation will be evaluated again anyway, so I will keep with if.

---

Now operations are only called from calculate, and only defined operations will call operations.